pkg/operator/ceph/csi/util_test.go

```diff
@@ -120,8 +120,8 @@ func Test_applyVolumeToPodSpec(t *testing.T) {
 		Param:     CSIParam,
 		Namespace: "foo",
 	}
-	// rbdplugin has 13 volumes by default
-	defaultVolumes := 13
+	// rbdplugin has 11 volumes by default
+	defaultVolumes := 11
```

Comment:
Can we add another csi test that checks for 13 volumes when log rotation is enabled, to help make sure log rotation is working right?